libosmscout-map/src/osmscoutmap/oss/Parser.cpp

```diff
@@ -61,7 +61,7 @@ void Parser::SynErr(int n)
 
 void Parser::SemErr(const char* msg)
 {
-  if (!state) {
```

Comment:
Note that Parser.cpp is generated from Parser.frame by coco, see generate.sh. You can change it here, but you must also apply the fix in he frame file, else Parser.cpp will be overwritten the next time, I change the parser...

Reply:
Ah ! okay